src/api/nodeRpcCall.js

```diff
@@ -1583,6 +1583,37 @@ const stakingBondExtra = async (value, walletAddress, callback) => {
   });
 }
 
+const getCongressMembers = async () => {
+  const api = await getApi();
+  const members = await api.query.council.members();
+  return members.map(m => m.toString());
+}
+
+const congressProposeLegislation = async (tier, index, legislationContent, walletAddress, callback) => {
+  const api = await getApi();
+  const injector = await web3FromAddress(walletAddress);
+  const congressmen = await api.query.council.members();
+  const threshold = parseInt(congressmen.length/2) + 1;
```

Comment:
in the future, they should be able to plan the threshold in advance, as councilproposals can be different in what referendum they spawn. But will add it as a new task